### PR TITLE
Stickertoimage

### PR DESCRIPTION
**Pull request by Zohre27**
"""
Avaible commands: .stickertoimage
"""
import datetime
from telethon import events
from telethon.errors.rpcerrorlist import YouBlockedUserError
from telethon.tl.functions.account import UpdateNotifySettingsRequest
from userbot.utils import admin_cmd

@borg.on(admin_cmd(pattern="stickertoimage ?(.*)"))
async def _(event):
    if event.fwd_from:
        return 
    if not event.reply_to_msg_id:
       await event.edit("```Reply to any user message.```")
       return
    reply_message = await event.get_reply_message() 
    if not reply_message.media:
       await event.edit("```Reply to sticker message```")
       return
    chat = "@DownloadStickersBot"
    sender = reply_message.sender
    if reply_message.sender.bot:
       await event.edit("```Reply to actual users message.```")
       return
    await event.edit("```Making a sticker to image```")
    async with event.client.conversation(chat) as conv:
          try:     
              response = conv.wait_event(events.NewMessage(incoming=True,from_users=658563529))
              await event.client.forward_messages(chat, reply_message)
              response = await response 
          except YouBlockedUserError: 
              await event.reply("```Please unblock me (@DownloadStickersBot) ```")
              return
          if response.text.startswith("Hi!"):
             await event.edit("```Can you kindly disable your forward privacy settings for good?```")
          else: 
             await event.delete()
             await event.client.send_message(event.chat_id, response.message)

### CODE REVIEW

**Comment by Adithyan1133-ctrl** [comment]
@Zohre27 Hoi

File: userbot/plugins/alive.py [hunk 1]
```diff
@@ -13,16 +13,13 @@
...
+    await alive.edit("**MY STATUS** \n`ZOHRE IS:` **✅ Alive**\n\n"
                      "`TELETHON VERSION:` **6.0.9**\n`Python:` **3.7.4**\n"
                      "`DATABASE STATUS:` **ALL NORMAL! WORKING FINE 🙂**\n`NO PROBLEM FOUND🔥!\n`"
                      "`CONNECTION PROVIDER:` ** AMAZON WEB SERVICES\n"
-                     "`CURRECT BOT LOCATION:` ** ASHBURN,UNITED STATES\n"
-                     "`DEPLOYING SERVICE BY: HEROKU INC.\n"
-                     "`IP:` 3.82.66.232\n"
+                     "`CURRECT BOT LOCATION:` ** ASHBURN,UNITED STATES\n" 
                      "`SYSTEM TYPE:` LINUX\n"
                      "`BASED ON:` UNIBORG\n"
```
Comment (Zohre27) [inline userbot/plugins/alive.py:22]: "`SATELLITE:` ⚡STARK SAT-2⚡\n"

**Comment by JyothisJayanth** [comment]
@Zohre27 You added the whole code as the Description 😂😬

**Comment by Jasur1224** [comment]
@JyothisJayanth .clone

**Comment by Zohre27** [comment]
> """
> Avaible commands: .stickertoimage
> """
> import datetime
> from telethon import events
> from telethon.errors.rpcerrorlist import YouBlockedUserError
> from telethon.tl.functions.account import UpdateNotifySettingsRequest
> from userbot.utils import admin_cmd
> 
> @borg.on(admin_cmd(pattern="stickertoimage ?(.*)"))
> async def _(event):
> if event.fwd_from:
> return
> if not event.reply_to_msg_id:
> await event.edit("`Reply to any user message.`")
> return
> reply_message = await event.get_reply_message()
> if not reply_message.media:
> await event.edit("`Reply to sticker message`")
> return
> chat = "@DownloadStickersBot"
> sender = reply_message.sender
> if reply_message.sender.bot:
> await event.edit("`Reply to actual users message.`")
> return
> await event.edit("`Making a sticker to image`")
> async with event.client.conversation(chat) as conv:
> try:
> response = conv.wait_event(events.NewMessage(incoming=True,from_users=658563529))
> await event.client.forward_messages(chat, reply_message)
> response = await response
> except YouBlockedUserError:
> await event.reply("`Please unblock me (@DownloadStickersBot) `")
> return
> if response.text.startswith("Hi!"):
> await event.edit("`Can you kindly disable your forward privacy settings for good?`")
> else:
> await event.delete()
> await event.client.send_message(event.chat_id, response.message)

**Comment by JyothisJayanth** [comment]
@Zohre27 Oof, Not again😅😂

**Pull request closed without merging**